configs/sites/hercules/packages.yaml

```diff
@@ -20,13 +20,12 @@ packages:
       prefix: /work/noaa/epic/role-epic/spack-stack/hercules/openmpi-4.1.5/gcc-11.3.1
       modules:
       - openmpi/4.1.5
-  python:
-    buildable: False
-    externals:
-    - spec: python@3.9.14 #+bz2+crypt+ctypes+dbm+lzma~nis+pyexpat~pythoncmd+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
-      prefix: /usr
 
 ### Modifications of common packages
+  # "Soft suggestion" in common/packages.yaml doesn't work, need 5.4.1 because of gcc version
+  py-pyyaml:
+    require:
+    - "@5.4.1"
```

Comment:
How critical is it for awscli to be the latest version? As I mentioned in my Slack message, I ran into the problem of conflicting awscli/py-pyyaml/py-botocore versions, and was able to resolve it by taking awscli and py-pyyaml back a version from the latest.

Reply:
Based on the slack conversation we had, shall we merge this as is or should I change py-pyyaml to 5.4.1 and set awscli accordingly (which version is it), and remove the extra settings for Hercules/Orion?

Reply:
If it works for all systems to set the versions in common/packages.yaml (py-pyyaml 5.4.1, awscli 1.27.84) then let's go ahead and do that.

Reply:
Done.